modules/persistent-storage-byok.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+// storage/container_storage_interface/persistent-storage-csi-azure.adoc
+// storage/container_storage_interface/persistent-storage-csi-ebs.adoc
+// storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+:_content-type: CONCEPT
+[id="byok_{context}"]
+= About "bring your own key" encryption
+
+The "Bring your own key" (BYOK) feature encrypts root volumes and the default storage class.
```

Comment:
"openshift node root volumes" just to clarify what we mean by root volumes.

Reply:
I am not sure we should keep the BOYK terminology, I think we should align with the installer doc and use "user managed encryption key" instead. Thoughts?

Reply:
+1 words like "Customer-managed(user-managed) encryption keys" is better than "BOYK".